srtcore/buffer_rcv.cpp

```diff
@@ -258,7 +258,7 @@ int CRcvBuffer::dropMessage(int32_t seqnolo, int32_t seqnohi, int32_t msgno, Dro
 {
     IF_RCVBUF_DEBUG(ScopedLog scoped_log);
     IF_RCVBUF_DEBUG(scoped_log.ss << "CRcvBuffer::dropMessage: seqnolo " << seqnolo << " seqnohi " << seqnohi
-        << ", msgno " << msgno << " m_iStartSeqNo " << m_iStartSeqNo);
+        << " msgno " << msgno << " actionOnExisting " << actionOnExisting << " m_iStartSeqNo " << m_iStartSeqNo);
```

Comment:
... this way here then:
```suggestion
        << " #" << msgno << " actionOnExisting " << actionOnExisting << " m_iStartSeqNo=%" << m_iStartSeqNo);
```